R/pkg/R/stats.R

```diff
@@ -19,7 +19,8 @@
 
 setOldClass("jobj")
 
-#' crosstab
+#' @title SparkDataFrame statistic functions
```

Comment:
Hi, @felixcheung . 
When I use `./create-docs.sh`, this breaks the page.
Should I do differently to generate the html page like yours?